### PR TITLE
Catch up with embulk v0.10

### PR DESCRIPTION
**Pull request by luongnhattruong**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```
Comment (dmikurube) [inline .github/workflows/build.yml]: We may want to use v2 for each. See https://github.com/embulk/embulk-util-guess/blob/v0.2.0/.github/workflows/check.yml#L7-L12

(Sorry that I've overlooked this in embulk-output-gcs.)

File: build.gradle [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
     id "jacoco"
+    id "signing"
+    id "maven-publish"
+    id "org.embulk.embulk-plugins" version "0.4.2"
 }
-import com.github.jrubygradle.JRubyExec
 
 repositories {
     mavenCentral()
     jcenter()
```
Comment (dmikurube) [inline build.gradle]: `jcenter()` should no longer be needed.

File: build.gradle [hunk 3]
```diff
@@ -35,27 +33,164 @@ shadowJar {
...
+
+                developers {
+                    developer {
+                        name = "Satoshi Akama"
+                        email = "satoshiakama@gmail.com"
+                    }
+                    developer {
+                        name = "John Luong"
+                        email = "jluong@treasure-data.com"
+                    }
```
Comment (dmikurube) [inline build.gradle]: From the Git log, let's go with this :

```suggestion
                    developer {
                        name = "Satoshi Akama"
                        email = "satoshiakama@gmail.com"
                    }
                    developer {
                        name = "Muga Nishizawa"
                        email = "muga.nishizawa@gmail.com"
                    }
                    developer {
                        name = "Naotoshi Seo"
                        email = "sonots@gmail.com"
                    }
                    developer {
                        name = "Robert Nguyen"
                        email = "ng.hung83@gmail.com"
                    }
                    developer {
                        name = "Hieu Duong"
                        email = "duongminhhieu89@gmail.com"
                    }
                    developer {
                        name = "John Luong"
                        email = "jluong@treasure-data.com"
                    }
```

File: build.gradle [hunk 4]
```diff
@@ -35,27 +33,164 @@ shadowJar {
...
 
 task classpath(type: Copy, dependsOn: ["jar", "shadowJar"]) {
     doFirst { file("classpath").deleteDir() }
-    from (configurations.runtime - configurations.provided
+    from (configurations.runtime
             + configurations.shadow
             - files(shadowJar.getIncludedDependencies())
             + files(shadowJar.archivePath))
     into "classpath"
 }
```
Comment (dmikurube) [inline build.gradle]: Do we still need this `classpath` task? I thought it's no longer needed.

**Review by dmikurube — COMMENTED** [review]
Although it may not be completed yet, let me leave comments as I once saw a review request.

One point is that: please do not forget `.lockfile`.

**Review by vietnguyen-td — APPROVED** [review]
LGTM

**Comment by luongnhattruong** [comment]
@dmikurube , connector work as expected on dev env. Please help me review again. Thank you.

**Review by dmikurube — APPROVED** [review]
May not be very appropriate to review my own commits, but at least I confirmed fot data exfiltrations.

**Pull request merged**